addons/link_tracker/models/link_tracker.py

```diff
@@ -132,7 +132,7 @@ def _get_title_from_url(self, url):
 
         return title
 
-    @api.constrains('url', 'campaign_id', 'medium_id', 'source_id')
+    @api.constrains('url', 'campaign_id', 'medium_id', 'source_id', 'label')
```

Comment:
shouldn't you make a migration script to ensure no "duplicated" link trackers exists ? (because you update the constraint)

Reply:
ah ok indeed :)